dipy/reconst/tests/test_shore_metrics.py

```diff
@@ -88,11 +88,9 @@ def test_shore_metrics():
     rtop_shore_pdf = asmfit.rtop_pdf()
     npt.assert_almost_equal(rtop_shore_signal, rtop_shore_pdf, 9)
     rtop_mt = multi_tensor_rtop([.5, .5], mevals=mevals)
-    npt.assert_equal(rtop_mt / rtop_shore_signal < 1.10 and
-                     rtop_mt / rtop_shore_signal > 0.95, True)
+    npt.assert_equal(1.10 > rtop_mt / rtop_shore_signal > 0.95, True)
```

Comment:
I think it would be clearer to split this up into two separate assertions. Also, `npt.assert_` is equivalent to `npt.assert_equal(foo, True)`, but slightly cleaner:
```suggestion
    npt.assert_(rtop_mt / rtop_shore_signal > 0.95)
    npt.assert_(rtop_mt / rtop_shore_signal < 1.10)
```